unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Transform/Utils/ECSTransformUtils.cs

```diff
@@ -48,6 +48,33 @@ public static bool TrySetParent(IParcelScene scene, IDCLEntity child, long paren
         return false;
     }
 
+    public static bool IsCircularParenting(IParcelScene scene, IDCLEntity entity, long parentId)
```

Comment:
We can maybe remove the `scene` parameter since `IDCLEntity` has a `scene` public property

Reply:
We are keeping scene and entities apart on the ecs7 development, so we can simplify and reduce dependencies for `IParcelScene` and `IDCLEntity` structures